libs/Microsoft.MixedReality.WebRTC.Unity/Assets/Plugins/arm64-v8a.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I think the folder name should be `ARM64` according to Unity conventions, no? I can't find again where I saw those though.

In any case those should move inside `Microsoft.MixedReality.WebRTC.Unity/` instead of being inside the `Plugins/` folder, which is the per-project folder for user plugins. I saw other products with native plugins (_e.g._ Photon) are putting all their files including DLLs inside a single folder. But I can do that part later.

Reply:
I think the convention might be `Plugins/Android` for Android native plugins, but not finding the docs for that now. I checked in the associated .meta files for each .aar, so they should be imported correctly in the build.

Reply:
I think `Plugins/Android` is for user's plugins. Probably Unity used to support only that, but now it seems you can put your plugins anywhere in `Assets`, so for third-party libraries it's best I think we keep all our files inside a single folder, that makes it much easier and cleaner for users importing multiple third-party libraries into their project and potentially also having their own plugins.